packages/react-sdk/src/stitches/stitches.ts

```diff
@@ -1,31 +1,30 @@
-import {
-  createStitches,
-  globalCss as globalCssImport,
-  type CSS,
-  css as createCss,
-} from "@stitches/core";
+import { createStitches, type CSS } from "@stitches/core";
 import type { Breakpoint } from "../css";
 
 let media = {};
 
-// @todo needs fixing
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
-let stitches: any;
+const createStitchesInstance = () => createStitches({ media });
+
+type StitchesInstance = ReturnType<typeof createStitchesInstance>;
```

Comment:
This way I "capture" the exact inferred type without supplying any generics manually etc.